### ISSUE TITLE
:attribute not replaced at beginning of string

### ISSUE THREAD

**Issue by solaris-isbc** [knuckleswtf/scribe#633]
We are using custom messages for validation rules (e. g. `validation.boolean`) to provide support for a non-english language. This works great except in cases where the message starts with `:attribute`. In this case it doesn't get replaced and displays `:attribute` in the generated documentation.
The reason for this seems to be that in ParsesValidationRules.php only ` :attribute ` (with a leading whitespace) is being replaced with a  value.
It would be expected, that even at the beginning of a string the placeholder `:attribute` is replaced.

**What happened?**
1. Created custom validation messages starting with `:attribute`
2. Generated documentation
3.  `:attribute` does not get replaced in the generated documentation

instead of...
Having `:attribute` replaced with value

**Screenshots and stack traces:**
![image](https://user-images.githubusercontent.com/37121713/224753829-f9fd259d-6143-4ea9-987f-cf1cc33c598b.png)

**My environment:**
 - PHP version: 8.1.14
 - Framework: Laravel
 - Laravel/Lumen version: 9.52.4 
 - Scribe version (from `composer show knuckleswtf/scribe`): 4.16.0

**My Scribe config (from `php artisan scribe:config:diff`):**
```
title => "[redacted]"
base_url => "[redacted]"
type => "laravel"
laravel.docs_url => "/docs/api"
try_it_out.enabled => false
auth.enabled => true
auth.default => true
auth.name => "Authorization"
auth.placeholder => "{AUTH_TOKEN}"
postman.enabled => false
strategies.queryParameters => added App\Support\Scribe\Strategies\QueryParameters\GetFromFormRequest, App\Support\Scribe\Strategies\EscapeParameterMustache: removed Knuckles\Scribe\Extracting\Strategies\QueryParameters\GetFromFormRequest
strategies.bodyParameters => added App\Support\Scribe\Strategies\BodyParameters\GetFromFormRequest, App\Support\Scribe\Strategies\EscapeParameterMustache: removed Knuckles\Scribe\Extracting\Strategies\BodyParameters\GetFromFormRequest
strategies.responses => added App\Support\Scribe\Strategies\EscapeResponseMustache
database_connections_to_transact.1 => "tenant"
```
**Additional info:**

**Comment by shalvah** [comment]
The space was intentional, to avoid unintentionally modifying things like "The field should be in the format key:attribute" (although probably an unlikely occurrence). Probably would be better to use a regex with word boundary (`\b:attribute`).

**Comment by shalvah** [comment]
This should have been fixed in 4.17.0
